patient-view-fe/src/components/anamnesis/anamnesis.component.tsx

```diff
@@ -2,9 +2,13 @@ import React from 'react'
 
 import {Anamnesis} from 'anamnesis'
 import 'anamnesis/dist/index.css'
```

Comment:
Why do you need the index.css from dist?

Reply:
Line 3 doesn't import css from library. I don't know why

Reply:
I will accept the PR in this way, but please have a look at the behavior.